airflow/settings.py

```diff
@@ -640,3 +640,5 @@ def initialize():
 
 # Prefix used to identify tables holding data moved during migration.
 AIRFLOW_MOVED_TABLE_PREFIX = "_airflow_moved"
+
+DAEMON_UMASK: str = conf.get('core', 'daemon_umask', fallback='0o077')
```

Comment:
```suggestion
DAEMON_UMASK: str = conf.get('core', 'daemon_umask')
```

nit: don't need this fallback.

Reply:
Oh this was sort of just to make typing happy -- otherwise this was `str | None`, but with the overload I added providing a fallback means it can't be None.

Reply:
I see. One day I'll start my crusade against these fallbacks littered everywhere, but today is not that day.

Reply:
My crusade is to remove writing defaults to `airflow.cfg` (but still have the defaults config live in code)

Reply:
"shields shall be splintered" some day ...